From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Mon, 30 Mar 2020 18:52:39 +0800
Subject: two-step: Center message text within labels if labels are centered

This makes a difference when you start rendering multiple lines in
a single label. For a single line label you won't see any difference.

Upstream: https://gitlab.freedesktop.org/plymouth/plymouth/-/merge_requests/103
---
 src/plugins/splash/two-step/plugin.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/plugins/splash/two-step/plugin.c b/src/plugins/splash/two-step/plugin.c
index 964855c..b8beacd 100644
--- a/src/plugins/splash/two-step/plugin.c
+++ b/src/plugins/splash/two-step/plugin.c
@@ -1876,6 +1876,9 @@ view_show_message (view_t     *view,
         ply_boot_splash_plugin_t *plugin = view->plugin;
         int x, y, width, height;
 
+        if (plugin->message_below_animation)
+                ply_label_set_alignment (view->message_label, PLY_LABEL_ALIGN_CENTER);
+
         ply_label_set_text (view->message_label, message);
         width = ply_label_get_width (view->message_label);
         height = ply_label_get_height (view->message_label);
